composer.json

```diff
@@ -77,5 +77,12 @@
     },
     "replace": {
         "zendframework/zend-expressive-authentication": "^1.1.0"
-    }
+    },
+    "repositories": [
+        {
+            "type": "git",
```

Comment:
```suggestion
            "type": "vcs",
```

Reply:
This is just temporary and should be removed. laminas/laminas-coding-standard:2.1.1 is just released with these changes.